app/main.py

```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person_data in people:
+        name = person_data["name"]
+        age = person_data["age"]
+        person = Person(name, age)
+
+        if "wife" in person_data and person_data["wife"] is not None:
```

Comment:
You could simplicate this check:
```suggestion
        if person_data.get("wife"):
```